lib/adapters/test-result/playwright.ts

```diff
@@ -34,6 +34,8 @@ export type PwtImageDiffError = ExtendedError<{snapshotName: string, diffCluster
 
 export type PwtNoRefImageError = ExtendedError<{snapshotName: string}>;
 
+export type PlaywrightImageFile = ImageFile & { relativePath: string };
```

Comment:
I don't want to pollute "ImageFile", which is used in Testplane and generally has no guaranteed "relativePath" property